content/en/blog/2023/istio-at-kubecon-na/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Istio at KubeCon North America 2023"
+description: A quick recap of Istio at KubeCon North America, at the McCormick Place in Chicago.
+publishdate: 2023-11-16
+attribution: "Faseela K, for the Istio Day Programme Committee"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+The open source and cloud native community gathered from the 6th to 9th November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```

Comment:
```suggestion
The open source and cloud native community gathered from the 6th to 9th of November in Chicago for the last KubeCon of 2023. The four-day conference, organized by
```

Reply:
Just caught this, too... `the 6th to the 9th of`.. I missed a the.

Reply:
Done